src/content/guides/code-splitting.md

```diff
@@ -114,6 +114,11 @@ The first of these two points is definitely an issue for our example, as `lodash
 
 The [`CommonsChunkPlugin`](/plugins/commons-chunk-plugin) allows us to extract common dependencies into an existing entry chunk or an entirely new chunk. Let's use this to de-duplicate the `lodash` dependency from the previous example:
 
+First, let's install this plugin: `syntax-dynamic-import`
+``` npm
+npm install --save-dev babel-plugin-syntax-dynamic-import
+```
```

Comment:
Hi @nicky-lau thanks for the contribution. IT's important to note users that this is only needed if they are transpiling their code with babel which might not be the case.

So I suggest changing this to a `W>` blockquote to inform users that **if** they are using Babel, they will need to include this so the Babel understands the dynamic import syntax, and remove it from the code below. Would you be able to do this?